Assets/Scripts/UI/TutorialText.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+using TMPro;
+
+[RequireComponent(typeof(TMP_Text))]
+public class TutorialText : MonoBehaviour
+{
+    private TMP_Text text;
+
+    private void Start()
+    {
+        text = GetComponent<TMP_Text>();
+    }
+
+    /// <summary>
+    /// Updates the text displayed by the Text object.
+    /// Call this function on an event.
+    /// </summary>
+    /// <param name="text">The new text for the </param>
```

Comment:
"The new text for the ...." 
For the what?

Reply:
Fixed comment